tracing/plugin/otlp_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+// TestNewExporter runs tests with different combinations of configuration for NewExporter function
+func TestNewExporter(t *testing.T) {
+
+	for _, testcase := range []struct {
+		input  OTLPConfig
+		output error
+	}{
+		//Test http/protobuf protocol, expect no error
+		{
+			input: OTLPConfig{Endpoint: "http://localhost:4318",
```

Comment:
Can we also have a test case to cover [this](https://github.com/fangn2/containerd/blob/1cdf2d245e03ff33d679bc94244af99453cd2084/tracing/plugin/otlp.go#L108) code path?